Prettier plugins not installing: [ERROR] Have not found plugin path for...

### Check for existing issues

- [X] Completed

### Describe the bug / provide steps to reproduce it

Install the XML or SQL extensions.
open a corresponding file associated with the extension.
Format the file.
The following error is added to the LOG:
`2024-10-10T15:32:11.971364+02:00 [ERROR] Have not found plugin path for "@prettier/plugin-xml" inside "/Users/me/Library/Application Support/Zed/prettier/node_modules"`
![xml-ext-missing](https://github.com/user-attachments/assets/f2ca761e-c336-4be8-8c51-30d26df0eb9e)




### Environment

macOS Sequoia 15.0.1
Zed 0.156.1

### If applicable, add mockups / screenshots to help explain present your vision of the feature

_No response_

### If applicable, attach your Zed.log file to this issue.

<details><summary>Zed.log</summary><pre>
<!-- Click below this line and paste or drag-and-drop your log-->

<!-- Click above this line and paste or drag-and-drop your log--></pre></details>

I'm running into this with the [zed-sql](https://github.com/nervenes/zed-sql/) extension failing to install `prettier-plugin-sql`.  I believe this may have broken for quite some time as I was able to reproduce the same issue with Zed from June 2024 (0.138.x).

Here is output from the most recent Nightly:
```
2024-10-13T14:16:37.228093-04:00 [INFO] installing extension sql latest version
2024-10-13T14:16:37.878036-04:00 [ERROR] No such file or directory (os error 2)
2024-10-13T14:16:37.882275-04:00 [INFO] rebuilt extension index in 11.878542ms
2024-10-13T14:16:37.884856-04:00 [INFO] extensions updated. loading 1, reloading 0, unloading 0
2024-10-13T14:16:37.917422-04:00 [ERROR] No such file or directory (os error 2)
2024-10-13T14:16:37.922831-04:00 [INFO] rebuilt extension index in 53.8235ms
2024-10-13T14:16:37.923626-04:00 [INFO] extensions updated. loading 1, reloading 0, unloading 0
2024-10-13T14:16:37.937321-04:00 [INFO] Initializing default prettier with plugins {}
2024-10-13T14:16:37.976697-04:00 [INFO] Initializing default prettier with plugins {"prettier-plugin-sql"}
2024-10-13T14:16:37.980689-04:00 [INFO] Initializing default prettier with plugins {"prettier-plugin-sql"}
2024-10-13T14:16:38.553592-04:00 [INFO] Installing default prettier and plugins: [("prettier-plugin-sql", "0.18.1"), ("prettier", "3.3.3")]
2024-10-13T14:16:38.802219-04:00 [INFO] Waiting for default prettier to install
2024-10-13T14:16:39.068435-04:00 [INFO] Initialized prettier with plugins: {"prettier-plugin-sql"}
2024-10-13T14:16:39.068542-04:00 [INFO] Starting prettier at path "/Users/peter/Library/Application Support/Zed/prettier"
2024-10-13T14:16:39.152451-04:00 [INFO] starting language server process. binary path: "/opt/homebrew/opt/node@20/bin/node", working directory: "/Users/peter/Library/Application Support/Zed/prettier", args: ["/Users/peter/Library/Application Support/Zed/prettier/prettier_server.js", "/Users/peter/Library/Application Support/Zed/prettier"]
2024-10-13T14:16:39.203652-04:00 [INFO] Started default prettier in "/Users/peter/Library/Application Support/Zed/prettier"
2024-10-13T14:16:39.203852-04:00 [ERROR] Have not found plugin path for "prettier-plugin-sql" inside "/Users/peter/Library/Application Support/Zed/prettier/node_modules"
```

--- a/crates/prettier/src/prettier.rs
+++ b/crates/prettier/src/prettier.rs
@@ -225,6 +225,8 @@ impl Prettier {
                                 prettier_plugin_dir.join("dist").join("index.mjs"),
                                 prettier_plugin_dir.join("dist").join("index.js"),
                                 prettier_plugin_dir.join("dist").join("plugin.js"),
+                                prettier_plugin_dir.join("src").join("plugin.js"),
+                                prettier_plugin_dir.join("lib").join("index.js"),
                                 prettier_plugin_dir.join("index.mjs"),
                                 prettier_plugin_dir.join("index.js"),
                                 prettier_plugin_dir.join("plugin.js"),

